tests/commands/open.ts

```diff
@@ -39,5 +39,11 @@ describe('Test clasp open function', () => {
     );
     expect(result.stdout).to.contain(LOG.OPEN_FIRST_PARENT(PARENT_ID[0]));
   });
+  it('open webapp with deploymentId page correctly', () => {
+    const result = spawnSync(
+      CLASP, ['open', '--webapp', '--deploymentId', 'abcd1234'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.not.contain('Open which deployment?');
```

Comment:
Could we change this to test the webapp is opened correctly. Currently this is testing the prompt for opening which deployment does not show, where the test case is describing a successful opening of the webapp to the correct deployment.

Reply:
Sure, I will slightly modify this test after merging PR #778

Reply:
Test is modfied